### ISSUE TITLE
Ranking leans towards short circuits?

### ISSUE THREAD

**Issue by cbizon** [NCATS-Gamma/robokop#466]
https://robokop.renci.org/a/72fc5b84-58f6-4f46-83ca-c5ee28d7ae31_f246c8bb-a271-4b0d-bd50-567ac8a073f7/

Question is going from glutamate receptors to memory loss using a COP-style query:

![image](https://user-images.githubusercontent.com/10744350/65690388-ecfa5880-e03c-11e9-96dc-73f366587d0e.png)

First answer:
![image](https://user-images.githubusercontent.com/10744350/65690460-174c1600-e03d-11e9-9fe0-fc6ef5a0c5b4.png)
the memory impairment / nervous system / neuron is all fine, but only connects to the gene part of the graph via neuron (not a real specific node)

Fourth answer:
![image](https://user-images.githubusercontent.com/10744350/65690720-a5280100-e03d-11e9-825d-68065a6e842d.png)

The left side of the graph is the same. And here the process is much more tightly bound to that side.  So  that should increase the score.  But the gene is less strongly connected.  In the original, the gene was more tightly coupled to (only) the cell.  Here, its still coupled, but less strongly.  So even though it's now also coupled to the process and the anatomy, it gets outweighed by the stronger connection to the cell.

I expect that this could be "solved" with modifying how edge scores are calculated?

**Comment by patrickkwang** [comment]
I imagine we could get the answer you want here by modifying how quickly the edge weights saturate. My concern is that there are other cases where we would prefer the parameters we have now. Before changing anything, I'm going to browse through all the issues and emails I can find and compile a list of queries that we want to go a certain way. There will be some bias toward things we don't/didn't like, but at least that will give us a few regression tests for parameter modifications.

**Comment by cbizon** [comment]
I agree with your concern.  Part of recording this one and a few others is to start to assemble a corpus that we can test against...
